src/init.cpp

```diff
@@ -699,7 +699,7 @@ static void ThreadImport(ChainstateManager& chainman, std::vector<fs::path> vImp
             if (!file)
                 break; // This error is logged in OpenBlockFile
             LogPrintf("Reindexing block file blk%05u.dat...\n", (unsigned int)nFile);
-            LoadExternalBlockFile(chainparams, file, &pos);
+            ::ChainstateActive().LoadExternalBlockFile(chainparams, file, &pos);
```

Comment:
I know I've asked the same question already, but I didn't get why this can't use `chainman.ActiveChainstate().Load...`.

This should simplify review because it will touch the same lines of code less often.

Also, it shouldn't make review not harder, because everyone knows that there is only one chainman (whether it is called `g_chainman` or `chainman` shouldn't matter). On top you added a runtime assertion, which validates this for all executed code paths in our tests:

```cpp
     assert(std::addressof(::ChainstateActive()) == std::addressof(ActiveChainstate()));

Reply:
Anyway, no big deal if you want to keep it that way. Will review regardless.